[2023-09-11T02:51:55.344+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v02.create_postgre_table manual__2023-09-11T02:51:53.890119+00:00 [queued]>
[2023-09-11T02:51:55.349+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v02.create_postgre_table manual__2023-09-11T02:51:53.890119+00:00 [queued]>
[2023-09-11T02:51:55.350+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-11T02:51:55.357+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): create_postgre_table> on 2023-09-11 02:51:53.890119+00:00
[2023-09-11T02:51:55.361+0000] {standard_task_runner.py:57} INFO - Started process 3634 to run task
[2023-09-11T02:51:55.363+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v02', 'create_postgre_table', 'manual__2023-09-11T02:51:53.890119+00:00', '--job-id', '344', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgre_operator.py', '--cfg-path', '/tmp/tmpz5jgycs_']
[2023-09-11T02:51:55.366+0000] {standard_task_runner.py:85} INFO - Job 344: Subtask create_postgre_table
[2023-09-11T02:51:55.400+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v02.create_postgre_table manual__2023-09-11T02:51:53.890119+00:00 [running]> on host 2e8147b3a465
[2023-09-11T02:51:55.446+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='andy_hsu' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v02' AIRFLOW_CTX_TASK_ID='create_postgre_table' AIRFLOW_CTX_EXECUTION_DATE='2023-09-11T02:51:53.890119+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-11T02:51:53.890119+00:00'
[2023-09-11T02:51:55.448+0000] {sql.py:274} INFO - Executing: 
            create table if not exists dag_runs (
                dt date,
                dag_id character varying,
                primary key (dt, dag_id)
            )
            
[2023-09-11T02:51:55.454+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-11T02:51:55.461+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-11T02:51:55.474+0000] {sql.py:418} INFO - Running statement: 
            create table if not exists dag_runs (
                dt date,
                dag_id character varying,
                primary key (dt, dag_id)
            )
            , parameters: None
[2023-09-11T02:51:55.484+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v02, task_id=create_postgre_table, execution_date=20230911T025153, start_date=20230911T025155, end_date=20230911T025155
[2023-09-11T02:51:55.535+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-11T02:51:55.548+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
